streams/src/test/java/org/apache/kafka/streams/state/internals/WindowStoreBuilderTest.java

```diff
@@ -183,18 +168,7 @@ public void shouldThrowNullPointerIfTimeIsNull() {
 
     @Test
     public void shouldThrowNullPointerIfMetricsScopeIsNull() {
-        reset(supplier);
-        expect(supplier.get()).andReturn(new RocksDBWindowStore(
```

Comment:
I assume this is removed because Mockito reported it as an unused stub or was there another reason?

Reply:
The relevant stubs are already setup in the `@Before` method and these aren't necessary.